google-cloud-build/ci-scripts/scripts/generate_source_target_docker_options.sh

```diff
@@ -1,14 +1,18 @@
+#!/bin/bash
+
 function generate_source_target_docker_options(){
   SCRIPT_DIR="$1"
   SOURCE_DOCKER_REPOSITORY="$2"
   SOURCE_TAG_PREFIX="$3"
   TARGET_DOCKER_REPOSITORY="$4"
   TARGET_TAG_PREFIX="$5"
   DOCKER_USERNAME="$6"
-  export TARGET_DOCKER_PASSWORD="$(cat secrets/DOCKER_PASSWORD)"
-  export SOURCE_DOCKER_PASSWORD="$(cat secrets/DOCKER_PASSWORD)"
-  TARGET_OPTIONS="$($SCRIPT_DIR/docker_options.sh 'target' $TARGET_DOCKER_REPOSITORY $DOCKER_USERNAME $TARGET_TAG_PREFIX)"
-  SOURCE_OPTIONS="$($SCRIPT_DIR/docker_options.sh 'source' $SOURCE_DOCKER_REPOSITORY $DOCKER_USERNAME $SOURCE_TAG_PREFIX)"
+  export TARGET_DOCKER_PASSWORD=""
+  TARGET_DOCKER_PASSWORD="$(cat secrets/DOCKER_PASSWORD)"
+  export SOURCE_DOCKER_PASSWORD=""
+  SOURCE_DOCKER_PASSWORD="$(cat secrets/DOCKER_PASSWORD)"
```

Comment:
Reason:
^-- SC2155: Declare and assign separately to avoid masking return values.